Python/ceval.c

```diff
@@ -752,7 +752,10 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
         goto exit_unwind;
     }
 
+// Jump here from ENTER_EXECUTOR and exit_trace.
+enter_tier_one:
     next_instr = frame->instr_ptr;
```

Comment:
The line setting `next_instr` was always here, and is (presumably) needed when we get here from `start_frame` or just via the top of the function. I only added the label so that the code that leaves Tier 2 (and has to set `next_instr`) can just jump here rather than doing `next_instr = frame->instr_ptr; goto resume_frame;` (which it was doing previously).

But this will have to change again when I do the work to avoid saving and restoring `stack_pointer` quite so much, as you requested here.